src/scripts/main.js

```diff
@@ -1,3 +1,68 @@
 'use strict';
 
-// write code here
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const container = document.querySelector('.container');
+const tr = document.querySelector('tr');
+const table = document.querySelector('.field');
+
+container.addEventListener('click', (e) => {
+  if (e.target === removeColumn) {
+    if (table.children[0].children[0].children.length >= 10) {
```

Comment:
```suggestion
    if (table.children[0].children[0].children.length >= 10) {
```
Цю перевірку на enabled/disabled для рядків/колонок можна прописати один раз після всіх if-ів.
Якщо після певних операцій рядків або колонок стає менше/більше, ніж потрібно - робити кнопку disabled/enabled